src/main/java/org/jenkinsci/plugins/casc/DescribableAttribute.java

```diff
@@ -23,37 +25,58 @@ public DescribableAttribute(String name, Class<? extends Describable> type) {
     public List<String> possibleValues() {
         final List<Descriptor> descriptors = Jenkins.getInstance().getDescriptorList(type);
         return descriptors.stream()
-                .map(d -> getSymbolName(d, type, d.getKlass().toJavaClass()))
+                .map(d -> getPreferredSymbol(d, type, d.getKlass().toJavaClass()))
                 .collect(Collectors.toList());
     }
 
-    public static String getSymbolName(Descriptor d, Class extensionPoint, Class target) {
+    /**
```

Comment:
NIT: it seems like you could move most of this logic into a separate new class like `SymbolFinder`, than would get that `getPreferredName()` method, and `getAliases()`. I guess that could make the code here smaller, and the feature more isolatedly testable.